api/services/sms/index.js

```diff
@@ -0,0 +1,17 @@
+const twilio = require('twilio');
+const { ACCOUNT_SID: accountSid, AUTH_TOKEN: authToken, PHONE_NUMBER } = require('config').TWILIO;
+
+class SmsService {
+  constructor(app) {
+    this.app = app;
+    this._client = twilio(accountSid, authToken);
+  }
+
+  sendMessage(body = 'Hola amigo', phone) {
```

Comment:
This is not valid 😛 , default arguments before required. Just remove the default one.